### ISSUE TITLE
prefer-modern-dom-apis: false positives with PixiJS

### ISSUE THREAD

**Issue by latin-1** [sindresorhus/eslint-plugin-unicorn#2372]
PixiJS's `Contaienr` has a '.removeChild(...)' API that should not be considered incorrect.

https://pixijs.download/release/docs/scene.Container.html#removeChild
```js
container.removeChild(child);
```

**Comment by fregante** [comment]
This is because unicorn doesn't know what `container` is. However the suggestion still applies since `child.removeFromParent()` exists in Pixi.js as well

**Comment by sindresorhus** [comment]
Nothing we can do about this.
